processing/data_collection/gazette/spiders/rs_canoas.py

```diff
@@ -0,0 +1,48 @@
+import json
+from datetime import date, datetime, timedelta
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class RsCanoasSpider(BaseGazetteSpider):
+    TERRITORY_ID = "4304606"
+    name = "rs_canoas"
+    START_DATE = date(2018, 5, 29)
```

Comment:
Can you use the `start_date` from the base class?

Reply:
Done